frontend/app/routes/employees/edit.js

```diff
@@ -0,0 +1,55 @@
+import Route from '@ember/routing/route';
+import RSVP from 'rsvp';
+
+export default Route.extend({
+  model(param){
+    return RSVP.hash({
+      employee: this.get('store').findRecord('employee',param.id),
+      picture: this.get('store').createRecord('picture'),
+      reportToEmployees: this.get('store').findAll('employee'),
+      departments: this.get('store').findAll('department'),
+      projects: this.get('store').findAll('project'),
+      roles: this.get('store').findAll('role'),
+      locations: this.get('store').findAll('location'),
+      skills: this.get('store').findAll('skill')
+    });
+  },
+  setupController(controller, model) {
+    this._super(controller, model);
+    controller.set('employee', model.employee);
+    let employeeList = [];
+    model.reportToEmployees.forEach(function (reportToEmployee) {
+      if(reportToEmployee.get('firstName')!=null){
+        if(reportToEmployee.get('firstName')!=model.employee.get('firstName')){
+          employeeList.push(reportToEmployee.get('firstName'));
```

Comment:
Reports to employee list only contains names to select, what if employee will have same names?